layouts/community/ortho_4x12/xyverz/rules.mk

```diff
@@ -1,7 +1,12 @@
 BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = yes       # Enable RGB Underglow functionality
 AUDIO_ENABLE = no           # Audio output on port C6
 
+ifeq ("$(KEYBOARD)","vitamins_included")
```

Comment:
Found that this may work better: 
```makefile
ifeq ("$(KEYBOARD)","vitamins_included")
  RGBLIGHT_ENABLE = no
else
  RGBLIGHT_ENABLE = yes
endif
```
It checks to make to see if see "vitamins_included" is part of the string.  So if there is a revision version, you can match that as well, using this.  For instance: 
```makefile
ifneq (,$(findstring planck,$(KEYBOARD)))
  ifneq (,$(findstring rev4,$(KEYBOARD)))
    BACKLIGHT_ENABLE = yes
    BACKLIGHT_BREATHING = yes
  else
    BACKLIGHT_ENABLE = no
    BACKLIGHT_BREATHING = no
  endif
  AUDIO_ENABLE = yes
endif
```
This will check for the planck keyboard. It will enable audio if it's a planck. But it will only enable backlight, if it's the rev4. 

I think this is the better way to go here, but either should be okay.